Source/WTF/wtf/text/StringImpl.cpp

```diff
@@ -849,7 +850,7 @@ size_t StringImpl::find(std::span<const LChar> matchString, size_t start)
         }
 
         size_t i = 0;
-        while (searchHash != matchHash || !equal(searchCharacters + i, matchString)) {
+        while (searchHash != matchHash || !equal(searchCharacters.data() + i, matchString)) {
```

Comment:
Kinda wish this was `subspace` instead, but I suppose that would require changing `equal`. Applies to other cases below.